src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/TestHelpers.java

```diff
@@ -183,5 +199,89 @@ public static NavigationUrls createDefaultNavigation() {
         return createNavigation(SUCCESS_URL, CANCEL_URL, ERROR_URL);
     }
 
+    public static AccountDetails createAccount(BigDecimal clientId) {
+
+        return AccountDetails.builder()
+                .fileRolePresent(true)
+                .accountId(BigDecimal.ONE)
+                .clientId(clientId)
+                .cardRegistered(true)
+                .universalId(UUID.randomUUID().toString())
+                .internalClientNumber(null)
+                .universalId(TestHelpers.CASE_1_STRING).create();
+
+    }
+
+    public static ReviewFilingPackage createFilingPackage() {
+
+        ReviewFilingPackage reviewFilingPackage = new ReviewFilingPackage();
+        reviewFilingPackage.setClientFileNo("CLIENTFILENO");
+        reviewFilingPackage.setFilingCommentsTxt(COMMENT);
+        reviewFilingPackage.setPackageNo(PACKAGE_NO);
+        reviewFilingPackage.setFirstName(FIRST_NAME);
+        reviewFilingPackage.setLastName(LAST_NAME);
+        reviewFilingPackage.setSubmittedDate(DateTime.parse("2020-05-05T00:00:00.000-07:00"));
+        reviewFilingPackage.setCourt(createReviewCourt());
+        reviewFilingPackage.setDocuments(Collections.singletonList(createDocument()));
+        reviewFilingPackage.setParties(Collections.singletonList(createParty()));
+        reviewFilingPackage.setPayments(Collections.singletonList(createPayment()));
+        return reviewFilingPackage;
+
+    }
+
+    public static ReviewCourt createReviewCourt() {
+
+        ReviewCourt reviewCourt = new ReviewCourt();
+        reviewCourt.setClassDescription(CLASS_DESCRIPTION);
+        reviewCourt.setCourtClass(COURT_CLASS);
+        reviewCourt.setDivision(DIVISION);
+        reviewCourt.setFileNumber(FILE_NUMBER);
+        reviewCourt.setLevel(LEVEL);
+        reviewCourt.setLevelDescription(LEVEL_DESCRIPTION);
+        reviewCourt.setLocationId(BigDecimal.ONE);
+        reviewCourt.setLocationName(LOCATION);
+        reviewCourt.setLocationDescription(LOCATION_DESCRIPTION);
+        reviewCourt.setParticipatingClass(PARTICIPATING_CLASS);
+        return reviewCourt;
+
+    }
+
+    public static Party createParty() {
+        return Party.builder()
+                .firstName(FIRST_NAME)
+                .lastName(LAST_NAME)
+                .middleName(MIDDLE_NAME)
+                .nameTypeCd(NAME_TYPE)
+                .partyTypeCd(ca.bc.gov.open.jag.efilingapi.api.model.Party.PartyTypeEnum.IND.getValue())
+                .roleTypeCd(ca.bc.gov.open.jag.efilingapi.api.model.Party.RoleTypeEnum.ABC.getValue())
```

Comment:
At some point, it would be nice to convert these partyTypeCd and roleTypeCd properties to be the actual enums instead of arbitrary String.  Will help with type-safety.

Reply:
good point

Reply:
We need some refactoring anyway